### PR TITLE
add bsblan documentation

### PR DESCRIPTION
**Pull request by liudger**
## Proposed change

First draft of documentation for bsblan integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/32375
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please don't remove the PR template next time 😉

File: source/_integrations/bsblan.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: BSBLan
+description: Instructions on how to integrate BSBLan device into Home Assistant.
+logo: bsblan.png
+ha_category: Climate
+ha_release: 0.106.2
```
Comment (klaasnicolaas) [inline source/_integrations/bsblan.markdown]:
```suggestion
ha_release: 0.107
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by liudger** [comment]
> Please don't remove the PR template next time 😉

My error. The first time I am doing a PR to home-assitant. 
With the integration I did automatically get an template. Now there was none?
But it could be that the PR was done wrongly

File: source/_integrations/bsblan.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+logo: bsblan.png
+ha_category: Climate
+ha_release: 0.107
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@liudger'
+---
+
+Integrates [BSBLan](https://github.com/fredlcore/bsb_lan) device into Home Assistant.
```
Comment (frenck) [inline source/_integrations/bsblan.markdown]: Could you add a bit of introduction text? Telling people what BSBLan is...

File: source/_integrations/bsblan.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+This integration can be configured using the integrations in the
+Home Assistant frontend.
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **BSBLan**.
+Fill in the IP address of the device in your network and if needed
+the port number. The default value should be 80.
+If you use a passkey or other authentication method on BSBLan device, you can't use the integration yet.
```
Comment (frenck) [inline source/_integrations/bsblan.markdown]:
```suggestion
If you use a passkey or other authentication method on the BSBLan device, you can't use the integration yet.
```

File: source/_integrations/bsblan.markdown [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+After completing the configuration flow, the BSBLan Climate integration will be
+available.
+
+For more documentation of the BSBLan device check the [manual](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
```
Comment (frenck) [inline source/_integrations/bsblan.markdown]:
```suggestion
For more documentation of the BSBLan device, check the [manual](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
```

File: source/_integrations/bsblan.markdown [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Configuration
+
+This integration can be configured using the integrations in the
+Home Assistant frontend.
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **BSBLan**.
+Fill in the IP address of the device in your network and if needed
```
Comment (frenck) [inline source/_integrations/bsblan.markdown]:
```suggestion
Fill in the IP address of the device in your network and, if needed,
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @liudger! 

Left a couple of small suggestions, could you please take a look? Thanks! 👍

File: source/_integrations/bsblan.markdown (hunk 2 above)
Comment (liudger) [reply source/_integrations/bsblan.markdown]: Hi frenck, 

Thanks for your time. I hope I fixed the documentation now. 
I still could need some help with the integration. Can't get passed the coverage error. Any suggestion are welcome!

found that the dev branch had a fix so I force pushed and now are all lights green

Comment (frenck) [reply source/_integrations/bsblan.markdown]: That is a different PR pending for review, will be picked up.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**